css/properties/background-clip.json

```diff
@@ -302,7 +305,9 @@
                 }
               ],
               "firefox": {
-                "version_added": "49"
+                "version_added": "49",
+                "partial_implementation": true,
+                "notes": "Does not work with <code>background-attachment:fixed</code>, in multi-layer backgrounds with other values, and in some other special cases: see <a href='https://bugzil.la/1656784'>bug 1656784</a>"
```

Comment:
```suggestion
                "notes": "Does not work with <code>background-attachment: fixed</code>, in multi-layer backgrounds with other values, and in some other special cases: see <a href='https://bugzil.la/1656784'>bug 1656784</a>"
```

Reply:
Can you add the specific bugs to the mentioned issues, to make it easier to check later if these have been resolved?

> `<code>background-attachment: fixed</code>`


https://bugzilla.mozilla.org/show_bug.cgi?id=1313757

> `in multi-layer backgrounds with other values`

https://bugzilla.mozilla.org/show_bug.cgi?id=1908386 ?

> `and in some other special cases`

Here you can keep the meta bug.

Reply:
Done, link text formatted according to CI rules requirements